packages/query/src/block-processor.ts

```diff
@@ -52,10 +56,16 @@ interface QueryClientProps {
   stakingTokenMetadata: Metadata;
 }
 
-const blankTxSource = new CommitmentSource({
+const BLANK_TX_SOURCE = new CommitmentSource({
   source: { case: 'transaction', value: { id: new Uint8Array() } },
 });
 
+const POSITION_STATES: PositionState[] = [
```

Comment:
Sort of the same here — extracting this constant from inside of a function, where it was being defined on every function call despite never changing.